<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>439</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/fileopen.png</normaloff>icons/fileopen.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout" rowminimumheight="6,6,6,6,6,6,6,6,6,6">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Шаблоны документов:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="cbxApprovalSert">
          <property name="text">
           <string>Свидетельство о поверке</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbTemplateApprovalCert">
          <property name="text">
           <string>C:\MCLinkReport\шаблон.xls</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btTemplateApprovalCert">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="statusTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="cbxApprovalProtocol">
          <property name="text">
           <string>Протокол поверки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbTemplateApprovalProtocol">
          <property name="text">
           <string>C:\MCLinkReport\шаблон.xls</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btTemplateApprovalProtocol">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="statusTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Папка XML:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbScanFolder">
          <property name="text">
           <string>C:\MCLinkReport\XML</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btScanFolder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Кнопка изменения папки XML</string>
          </property>
          <property name="statusTip">
           <string>Кнопка изменения папки XML</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Папка протоколов:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbDestFolder">
          <property name="text">
           <string>C:\MCLinkReport\Excel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btDestFolder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Кнопка изменения папки Excel</string>
          </property>
          <property name="statusTip">
           <string>Кнопка изменения папки Excel</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QCheckBox" name="cbxCalSert">
          <property name="text">
           <string>Сертификат калибровки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbTemplateCalCert">
          <property name="text">
           <string>C:\MCLinkReport\шаблон.xls</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btTemplateCalCert">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="statusTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCheckBox" name="cbxCalProtocol">
          <property name="text">
           <string>Протокол калибровки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbTemplateCalProtocol">
          <property name="text">
           <string>C:\MCLinkReport\шаблон.xls</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btTemplateCalProtocol">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="statusTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="cbxError">
          <property name="text">
           <string>Извещение о непригодности</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbTemplateError">
          <property name="text">
           <string>C:\MCLinkReport\шаблон.xls</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btTemplateError">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="statusTip">
           <string>Кнопка изменения шаблона</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10"/>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Пути:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="edApprovalProtocolNum"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="edApprovalCertNum"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="edErrorNum"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="edCalProtocolNum"/>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="edCalCertNum"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Номера документов:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="edTemplateReserved">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btSaveSettings">
        <property name="text">
         <string>Сохранить настройки</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chbAutoOpen">
        <property name="toolTip">
         <string>Автооткрытие протокола</string>
        </property>
        <property name="statusTip">
         <string>Автооткрытие протокола</string>
        </property>
        <property name="text">
         <string>Автоматическое открытие протоколов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbAutoDelXML">
        <property name="text">
         <string>Автоудаление XML</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="exitAction"/>
   <addaction name="startAction"/>
   <addaction name="stopAction"/>
  </widget>
  <action name="exitAction">
   <property name="icon">
    <iconset>
     <normaloff>icons/application-exit.png</normaloff>icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="statusTip">
    <string>Выход</string>
   </property>
  </action>
  <action name="startAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/player_play.png</normaloff>icons/player_play.png</iconset>
   </property>
   <property name="text">
    <string>Запуск автосохранения</string>
   </property>
   <property name="statusTip">
    <string>Запуск автосохранения</string>
   </property>
  </action>
  <action name="stopAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/player_stop.png</normaloff>icons/player_stop.png</iconset>
   </property>
   <property name="text">
    <string>Остановка автосохранения</string>
   </property>
   <property name="statusTip">
    <string>Остановка автосохранения</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
